elyra/pipeline/component.py

```diff
@@ -153,7 +153,7 @@ class Component(object):
 
     def __init__(self, id: str, name: str, description: Optional[str], source_type: str,
                  source: str, runtime: Optional[str] = None, op: Optional[str] = None,
-                 properties: Optional[List[ComponentParameter]] = None,
+                 category: str = None, properties: Optional[List[ComponentParameter]] = None,
```

Comment:
```suggestion
                 category: Optional[str] = None, properties: Optional[List[ComponentParameter]] = None,
```
Should this parameter be named `category_id` or `category_name` or leave alone?  Not sure what the best answer is.    Since these also have a label, I think `category_id` would be the least confusing.

Reply:
I agree on `category_id`!